wallet/src/repository/wallet/mod.rs

```diff
@@ -1086,7 +1086,8 @@ fn build_tally_report(
                 .collect::<Result<HashMap<PublicKeyHash, model::Reveal>>>()?;
 
             // Set not `in_consensus` reveals
-            for pkh in &tally.slashed_witnesses {
+            // TODO: Review that part for wallet guys (now out of consensus is no the same that slashed)
```

Comment:
@mariocao, @aesedepece could you review that part?

Reply:
![image](https://user-images.githubusercontent.com/1316439/83630907-e5605b80-a59c-11ea-8ce8-3ab378e6672f.png)

Reply:
seems ok... maybe it could have some indirect consequence in the way `non-conensus` reveals are shown in Sheikah

Reply:
perfect, but in that case it could be solved in a little fix in the future if we notice that